src/src/PluginStructs/P166_data_struct.cpp

```diff
@@ -124,6 +124,9 @@ bool P166_data_struct::plugin_write(struct EventStruct *event,
               if ((nChannel == 1) || (nChannel == 2)) {
                 setUserVarAndLog(event, 1, voltValue, fValue, subcommand);
               }
+
+              sendData(event); // Send out data for events and to controllers
```

Comment:
Why not schedule a `PLUGIN_READ`?
No idea if this may cause some stack overflow issues with some controller.

Reply:
Fixed.